From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for drm_gem.h
Patch-mainline: Never, kABI workaround
References: git-fixes

The patch
  patches.suse/drm-gem-Test-for-imported-GEM-buffers-with-helper.patch
added a new header inclusion that broke kABI (symbols got defined).

Wrap it with standard __GENKSYMS__ check as a workaround.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 include/drm/drm_gem.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/drm/drm_gem.h
+++ b/include/drm/drm_gem.h
@@ -35,7 +35,9 @@
  */
 
 #include <linux/kref.h>
+#ifndef __GENKSYMS__
 #include <linux/dma-buf.h>
+#endif
 #include <linux/dma-resv.h>
 #include <linux/list.h>
 #include <linux/mutex.h>
